pom.xml

```diff
@@ -572,5 +572,97 @@
         <skipITs>false</skipITs>
       </properties>
     </profile>
+    <profile>
+      <id>devsite-apidocs</id>
+      <activation>
+        <property>
+          <name>devsite.template</name>
```

Comment:
Where/what sets this property? How would a developer set this if they needed to run this profile themselves?